osu.Game/Users/UserActivity.cs

```diff
@@ -90,9 +90,16 @@ public Editing(IBeatmapInfo info)
             public override string Status => @"Editing a beatmap";
         }
 
-        public class Spectating : UserActivity
+        public class Watching : UserActivity
         {
-            public override string Status => @"Spectating a game";
+            protected virtual string Verb => @"Watching";
+
+            public override string Status => @$"{Verb} a game";
+        }
+
+        public class Spectating : Watching
+        {
+            protected override string Verb => @"Spectating";
```

Comment:
If you're going to implement these, you probably want to include details such as the player and beatmap information? Check all the existing implementations (`InGame` and derived variants).